capella_model_explorer/components.py

```diff
@@ -284,6 +284,10 @@ def report_loader(
     template_id: str,
     model_element_uuid: str = "",
 ) -> t.Any:
+    resource_info = state.model.info.resources["\x00"]
+    branch = resource_info.branch
+    hash = resource_info.rev_hash
+    vary_value = f"{branch}-{hash}"
```

Comment:
The branch and hash of the model are not sufficient as cache identifier. What happens if the fastHTML code changes? What happens when there is an update to a template? You should also consider the version / hash of the template and the package version.